cirq-core/cirq/circuits/circuit_operation.py

```diff
@@ -85,6 +85,9 @@ class CircuitOperation(ops.Operation):
     """
 
     _hash: Optional[int] = dataclasses.field(default=None, init=False)
+    _final_measurement_key_objs: Optional[AbstractSet[value.MeasurementKey]] = dataclasses.field(
```

Comment:
Why "final"? If the purpose is to distinguish from the magic method, `_cached_measurement_key_objs` makes more sense to me

Reply:
I used `final` to indicate that this set is after all remappings and repetition ids are applied. `cached` works as well. Updated